src/components/Loader/Loader.tsx

```diff
@@ -1,7 +1,8 @@
 import './Loader.scss';
+import { memo } from 'react';
 
-export const Loader = () => (
+export const Loader = memo(() => (
```

Comment:
where needed, not everywhere
there is no benefit in such cases, also you don't accept any props, so it's useless